packages/data-service/src/data-service.ts

```diff
@@ -947,8 +947,8 @@ class DataServiceImpl extends WithLogContext implements DataService {
             $replaceRoot: {
               newRoot: {
                 $mergeObjects: [
-                  { $first: '$stats' },
-                  { $first: '$indexSizes' },
+                  { $arrayElemAt: ['$stats', 0] },
+                  { $arrayElemAt: ['$indexSizes', 0] },
```

Comment:
Do you want to keep `$indexSizes`? As mentioned on Slack, we don’t seem to be using it really and we could remove the `$facet` and `$replaceRoot` without it. On the other hand, if we ever intend to use it again, having this pipeline already written would of course be nice. Looks good either way!